packages/running/src/index.tsx

```diff
@@ -323,56 +228,54 @@ export class RunningSessions extends ReactWidget {
   /**
    * Construct a new running widget.
    */
-  constructor(options: RunningSessions.IOptions) {
+  constructor(managers: IRunningSessionManagers) {
```

Comment:
Note: This change seems to require a major version bump of the owning package.

Reply:
I think you're right. I've opened https://github.com/jupyterlab/jupyterlab/issues/7219 to track this.